dozer-sql/Cargo.toml

```diff
@@ -22,6 +22,7 @@ hashbrown = "0.13"
 ahash = "0.8.3"
 bloom = "0.3.2"
 enum_dispatch = "0.3.11"
+proptest = "1.1.0"
```

Comment:
Can we move this to `dev-dependencies `?

Reply:
can